File: 2114. Maximum Number of Words Found in Sentences/2114. Maximum Number of Words Found in Sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        maxx=0        
        for i in range(len(sentences)):
            count=0
            for j in sentences[i].split():
                count+=1
            if count>maxx:
                maxx=count
        return maxx

File: 2114. Maximum Number of Words Found in Sentences/2114. Maximum Number of Words Found in Sentences.c
int mostWordsFound(char** sentences, int sentencesSize) {
    int max=0;
    int count;
    for(int i=0; i<sentencesSize; i++){
        count = wordCount(sentences[i]);
        if(count>max){
            max=count;
        }
    }
    return max;    
}
int wordCount(char* sentences){
    int count=0;
    for(int j=0; sentences[j]!= '\0'; j++){
        if(sentences[j]==' '){
            count++;
        }
    }
    return count+1;
}
